__react/template/src/tests/testclient/testchannels/$$channel$$.ts.js

```diff
@@ -0,0 +1,84 @@
+import { File } from '@asyncapi/generator-react-sdk';
```

Comment:
What is a difference between this and [this](https://github.com/asyncapi/ts-nats-template/pull/23/files#diff-6673dc75614475b27a7498b9dfcbc24e4fba0253c0eddf27c1854e3dee19fe03R1) file?

Reply:
Maybe you should create separate file with code, and then import and use in those two files without redeclaration two times?

Reply:
The difference is that it is a mirror channel, I am afraid splitting the code any more will be more confusing even though more maintainable 😕 I mean the only things that change are the messages are switched and the import path is different. I could give it a shot and then we will see 😄

Reply:
ok 👍🏼